x-pack/plugins/ml/public/application/components/chart_tooltip/chart_tooltip_service.test.ts

```diff
@@ -1,21 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-
-import { getChartTooltipDefaultState, mlChartTooltipService } from './chart_tooltip_service';
-
-describe('ML - mlChartTooltipService', () => {
-  it('service API duck typing', () => {
-    expect(typeof mlChartTooltipService).toBe('object');
-    expect(typeof mlChartTooltipService.show).toBe('function');
-    expect(typeof mlChartTooltipService.hide).toBe('function');
-  });
-
-  it('should fail silently when target is not defined', () => {
-    expect(() => {
-      mlChartTooltipService.show(getChartTooltipDefaultState().tooltipData, null);
-    }).not.toThrow('Call to show() should fail silently.');
-  });
-});
```

Comment:
Not the most extensive set of tests but is there any reason not to keep them as a basis?

Reply:
I will write another set of tests

Reply:
Done in https://github.com/elastic/kibana/pull/64056/commits/ce66cd2cf23f83cda2221156dd978bf4bb9dadba